src/rpc/mempool.cpp

```diff
@@ -959,6 +977,8 @@ static RPCHelpMan submitpackage()
                             replaced_txids.insert(ptx->GetHash());
                         }
                     }
+                } else {
+                    NONFATAL_UNREACHABLE();
```

Comment:
nit/followup: This could instead be a compile-time check if the if/else blocks are converted to a switch statement on result type

Reply:
done